db/migrate/20220203164307_remove_shortened_quarantine_filters.rb

```diff
@@ -5,15 +5,19 @@ def up
       relevant_filters.each do |uf|
         contents = JSON.parse(uf[:contents])
         contents = migrate_advanced_filter_contents(contents)
-        uf.update!(contents: contents.to_json)
+        if contents.empty?
```

Comment:
Not really sure why but now I'm seeing this happen:

<img width="1180" alt="Screen Shot 2022-02-04 at 4 00 41 PM" src="https://user-images.githubusercontent.com/35042815/152602521-e68aeed4-7958-4c88-8c5e-528607894071.png">

Reply:
This is post migration where the top filter was one of the quarantine ones

Reply:
I just ran a clean migration and am no longer seeing this, so my dev environment must have been in a weird state